azure/Kqlmagic/kql_magic.py

```diff
@@ -194,6 +197,19 @@ def validate_cloud(self, cloud):
             )
 
 
+    @validate("login_code_destination")
+    def _valid_value_login_code_destination(self, proposal):
+        dest = proposal["value"]
+        if dest != "browser":
+            if not self.check_mailto(dest):
+                message = "The 'login_code_destination' trait of a {0} instance {1}".format(Constants.MAGIC_CLASS_NAME, str(e))
+                raise TraitError(message)
+        return proposal["value"]
```

Comment:
not suer it can be done, but I suggest you return a lowercase of proposal["value"] to make it case incensitive